README.md

```diff
@@ -10,7 +10,7 @@ This library is part of the [**WP on Rails**](https://github.com/wponrails) proj
 
 To prepare a WP installation to be used in a **WP on Rails** architecture the [**wp-relinquish**](https://github.com/wponrails/wp-relinquish) plugin is provided. It provides a means to configure WP actions to send notifications as webhook calls to a Rails application equipped with **wp-connector**.
 
-When using **WP on Rails** the content's *master data* resides in WP's database, as that's where is it created and modified.  The Rails application that is connected to WP stores merely a copy of the data, a cache, on the basis of which the public requests are served.
+When using **WP on Rails** the content"s *master data* resides in WP"s database, as that"s where is it created and modified.  The Rails application that is connected to WP stores merely a copy of the data, a cache, on the basis of which the public requests are served.
```

Comment:
@Marthyn  Here single quotes are actually valid. Regardless of the rubocop setting. Did rubocop report this as wrong inside a .md file?